app/controllers/complaints_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ComplaintsController < ApplicationController
+  @@hses_issues = FakeData::ApiResponse.generate_hses_issues_response[:data]
```

Comment:
[suggestion] I think I like the method of saving these, but I think I don't like the controller being responsible for holding onto the data. What if we added a `FakeData::ApiResponse.hses_issues_response` method that memoized the return value of `generate_hses_issues_response`?

I think that would also clarify where in the test stack is the right place to test the behavior.

Reply:
Ope yep, that is a very good call